src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -943,7 +945,8 @@ void Transformations::MainSnippets(void) {
         if (matmul->get_transpose_a() || matmul->get_transpose_b())
             return false;
         if (in_type0 == ov::element::i8)
```

Comment:
I think we missed something here... Probably we should check for both `i8` and `u8` here. Could you please verify this moment?

Reply:
I looked through onednn code & docs, and it looks like they do support both i8 and u8 on the firs input, [here](https://github.com/openvinotoolkit/oneDNN/blob/8aec0b5b29d61bd27e561bf1c8f300e0d56d1ac1/src/cpu/x64/brgemm/brgemm_utils.cpp#L52) for example 🤔 

@v-Golubev you are an expert in the low-precision pipeline. Do you think it can be for a reason for i8 support only? 

Anyways, we should not change it in this PR, since this change needs a separate validation do distinguish from avx2 impacts.
 I'll check tomorrow if there are any models in the cache that have u8 A input. Let's start with that.

Reply:
I just think that this condition was added by me a one year ago and I mixed up first and second inputs: Probably, I thought that only second input may have different precisions... But not 😄 

> I'll check tomorrow if there are any models in the cache that have u8 A input. Let's start with that.
If it's possible, it will be great! Thank you in advance!

Reply:
> @v-Golubev you are an expert in the low-precision pipeline. Do you think it can be for a reason for i8 support only?

I looked through this lambda, and it seems like we support u8 on the first input without additional requirements (we just skip all ifs and return true). As for i8 precision, we must probably have vnni instructions because compensations are computed in i8i8 precisions configuration.